This file is part of mingw-cross-env.
See doc/index.html for further information.

Hack to use pkg-config with static linking.

--- a/libidn.pc.in
+++ b/libidn.pc.in
@@ -19,5 +19,5 @@
 Description: IETF stringprep, nameprep, punycode, IDNA text processing.
 URL: http://www.gnu.org/software/libidn/
 Version: @VERSION@
-Libs: -L${libdir} -lidn
+Libs: -L${libdir} -lidn -liconv
 Cflags: -I${includedir}
